code/modules/cargo/packs.dm

```diff
@@ -1136,6 +1136,18 @@
 	for(var/i in 1 to 49)
 		new /mob/living/simple_animal/butterfly(.)
 
+/datum/supply_pack/organic/critter/crab
+	name = "Crab Rocket"
+	cost = 5000
+	contains = list(/mob/living/simple_animal/crab)
+	crate_name = "look sir free crabs"
```

Comment:
>free crabs
>`cost = 5000`
unplayable.

Reply:
the rocket is the part that costs money

Reply:
^

Reply:
crates are worth 500
correct manifests are worth 200
make it cost 700 credits to order

also who puts their crabs in a rocket? Drop crate or bust